### ISSUE TITLE
PHP Warnings with Hathor

### ISSUE THREAD

**Issue by mbabker** [joomla/joomla-cms#12777]
```sh
Warning: strrpos(): Offset is greater than the length of haystack string in libraries/cms/html/html.php on line 373
Call Stack
#	Time	Memory	Function	Location
1	0.0014	372104	{main}( )	.../index.php:0
2	0.1487	4188352	JApplicationCms->execute( )	.../index.php:51
3	0.5115	11004544	JApplicationAdministrator->render( )	.../cms.php:271
4	0.5116	11010216	JApplicationCms->render( )	.../administrator.php:429
5	0.5117	11010592	JDocumentHtml->parse( )	.../cms.php:1090
6	0.5117	11010592	JDocumentHtml->_fetchTemplate( )	.../html.php:457
7	0.5118	11010688	JDocumentHtml->_loadTemplate( )	.../html.php:642
8	0.5128	11054408	require( 'administrator/templates/hathor/cpanel.php' )	.../html.php:580
9	0.5191	11060560	JHtml::_( )	.../cpanel.php:68
10	0.5192	11061008	JHtml::call( )	.../html.php:105
11	0.5192	11061832	JHtml::stylesheet( )	.../html.php:233
12	0.5192	11062264	JHtml::includeRelativeFiles( )	.../html.php:630
13	0.5193	11062936	strrpos ( )	.../html.php:373
```

**Comment by andrepereiradasilva** [comment]
probably related to one of my PR. will check

**Comment by andrepereiradasilva** [comment]
i don't have this warning
@mbabker are you using php 7.1?

Actually that issue is there for ever my PR only make it the see the light of day :)

The error is saying that 4 is actually bigger than the whole string.
https://github.com/joomla/joomla-cms/blob/staging/libraries/cms/html/html.php#L373

This is the case for ie8.css (ie8 = 3 characters) in hathor template
https://github.com/joomla/joomla-cms/blob/staging/administrator/templates/hathor/cpanel.php#L67

i think i solved it in https://github.com/joomla/joomla-cms/pull/12780, but since i don't have the warning i can't confirm. please confirm @mbabker

**Comment by andrepereiradasilva** [comment]
please test https://github.com/joomla/joomla-cms/pull/12780

**Comment by andrepereiradasilva** [comment]
ok i was able to reproduce it. you need debug enabled and debug plugin enabled and go to control panel in hathor.
Anyway my PR fixes it.

**Comment by brianteeman** [comment]
Closed as we have a PR for testing #12780
